api/s2n.h

```diff
@@ -333,6 +333,39 @@ S2N_API
 extern int s2n_connection_set_read_fd(struct s2n_connection *conn, int readfd);
 S2N_API
 extern int s2n_connection_set_write_fd(struct s2n_connection *conn, int writefd);
+
+/**
+  * Gets the assigned fd for the read channel of an s2n connection.
+  *
+  * In case the s2n connection was set to managed io on the read channel
+  * via s2n_connecion_set_fd or s2n_connection_set_read_fd.
```

Comment:
Was there supposed to be more to this sentence? I'm not sure what it's trying to say

Reply:
removed